app/assets/javascripts/ajax_events.js

```diff
@@ -6,10 +6,4 @@
     .on('ajaxComplete ajax:complete pjax:end', function(event, xhr, status){
       $('#ajax-in-progress').remove();
     })
-    .on('ajax:error', function(event, xhr, status, error){
-      alert('Request failed - ' + status);
-      console.error(error);
-      // report error to errorception in async manner
-      setTimeout(function(){ throw(error) }, 1);
```

Comment:
Besides the fact that the comment mentions it, how's this `alert` and `throw(error)` related to errorception? 🤔Maybe it's still used by bugsnag?

Reply:
I'm not sure, it's the 1st time I deal with either bugsnag or errorception, but I don't think is used by bugsnag, otherwise tests like [this](https://github.com/3scale/porta/blob/master/test/unit/events/importers/alert_importer_test.rb#L99) or [this](https://github.com/3scale/porta/blob/edf4cfcb170c6a6613b652796f6fab6f70674d91/test/unit/event_store/repository_test.rb#L104) would fail

Reply:
And I think the `alert` is not related to anything, is just for the user , I guess

Reply:
Maybe @didierofrivia knows

Reply:
Only the `setTimeout` (line 13) needs to be removed.
Basically bugsnag or errorception catch the error thrown, so here it will catch a failed ajax request.

Reply:
Thanks @hallelujah